apps/site/assets/ts/models/alert.ts

```diff
@@ -182,3 +182,10 @@ export const hasAnActiveDiversion = (
   stopId: StopId,
   alerts: Alert[]
 ): boolean => alertsByStop(alerts, stopId).some(isActiveDiversion);
+
+export const isGlobalBannerAlert = (alert: Alert): boolean => {
+  return (
+    alert.header.split("\n").join(" ") ===
+    document.getElementById("global-banner-description")?.innerText
```

Comment:
I think we do some link creation on the inner text so this might not work matching when the alert has a link in it